Forms/MainForm.cs

```diff
@@ -774,7 +774,7 @@ public void InitializeSettings()
             _settings.FixEponaSword = true;
             _settings.ShortenCutscenes = true;
             _settings.QuickTextEnabled = true;
-            _settings.UpdateChests = true;
+            _settings.UpdateChests = false;
```

Comment:
Revert this. And don't try to sneak things in by omitting them from the pull request description.

Reply:
I was not trying to sneak anything in, I just forgot a small detail. This should not be on by default at all.

Reply:
Please revert this change as we discussed.